src/core/fonts.js

```diff
@@ -3216,10 +3216,14 @@ var Font = (function FontClosure() {
         };
       }
 
-      var fontChar =
-        typeof fontCharCode === "number"
-          ? String.fromCodePoint(fontCharCode)
-          : "";
+      let fontChar = "";
```

Comment:
Is this method called often? If it's often, it may be beneficial to leave it undefined here and only set it to the empty string in the exception cases to avoid unnecessary string allocation in the common case when the code point is valid.

Reply:
> Is this method called often?

Yes, but comparing the run-time of the tests here to other PRs there really doesn't seem to be any noticeable slowdowns overall (I *think* browsers may be able to optimize/re-use strings internally).

> and only set it to the empty string in the exception cases

While I did consider that, it's not clear to me that it's entirely necessary here (also the code would become a bit "messy" in that case IMHO).
Also, you're still having to allocate the variable (albeit implicitly to `undefined`) which is probably why it doesn't seem to matter in practice.